public/app/features/dashboard-scene/utils/utils.ts

```diff
@@ -113,3 +120,19 @@ export function getMultiVariableValues(variable: MultiValueVariable) {
     texts: Array.isArray(text) ? text : [text],
   };
 }
+
+export function getQueryRunnerFor(sceneObject: SceneObject | undefined): SceneQueryRunner | undefined {
```

Comment:
I see this function in 3rd pr, not sure if this is the same one ;)

Reply:
@dprokop I needed it in multiple PRs so will have to deal with the merge conflicts